packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -99,6 +100,26 @@ it('should overflow when there is not enough space', () => {
   expect(breadcrumbs[2].textContent).toEqual('Item 2');
 });
 
+it('should handle overflow when overflowButton is specified', () => {
+  const onClick = jest.fn();
+  useOverflowMock.mockReturnValue([jest.fn(), 2]);
+  const { container } = renderComponent({
+    overflowButton: (visibleCount) => (
+      <IconButton onClick={onClick(visibleCount)}>
+        <SvgMoreSmall />
+      </IconButton>
+    ),
+  });
+
+  expect(container.querySelector('.iui-breadcrumbs')).toBeTruthy();
+  expect(container.querySelector('.iui-breadcrumbs-list')).toBeTruthy();
+
+  const breadcrumbs = container.querySelectorAll('.iui-breadcrumbs-item');
+  expect(breadcrumbs.length).toEqual(3);
+  fireEvent.click(breadcrumbs[1]);
+  expect(onClick).toHaveBeenCalledTimes(1);
```

Comment:
You can also check `toHaveBeenCalledWith`

Reply:
Added